addon/mixins/registration-actions.js

```diff
@@ -22,6 +29,15 @@ export default Ember.Mixin.create({
         }
         return currentMetadata;
     },
+    /**
+    * Build the request payload used when creating a registration from a draft.
+    *
+    * @method _generateRegistrationPayload
+    * @param {String} draft ID of the draft registration
+    * @param {String} registrationChoice Either "immediate" or "embargo".
+    * @param {Date} liftEmbargo, if registrationChoice === "embargo". liftEmbargo should be the date to lift the embargo.
```

Comment:
Unbelievably trivial: there are a few trailing commas that are probably not needed throughout. (and one period that could be a comma)